packages/workflow/src/errors/configuration.error.ts

```diff
@@ -0,0 +1,9 @@
+import { UserError } from './base/user.error';
+
+/**
+ * Error that indicates the user provided invalid configuration.
+ *
+ * Default level: info
+ * Default shouldReport: false
+ */
+export class ConfigurationError extends UserError {}
```

Comment:
wondering if this sub-classing is worth is if this class gets used only once. Could we maybe just use `UserError` everywhere for now, and create this subclass once we have enough places in code to justify having another error type for?